superset/assets/cypress/integration/sqllab/sourcePanel.js

```diff
@@ -0,0 +1,60 @@
+import { selectResultsTab } from './sqllab.helper';
+
+export default () => {
+  describe('SqlLab datasource panel', () => {
+    beforeEach(() => {
+      cy.login();
+      cy.server();
+      cy.visit('/superset/sqllab');
+    });
+
+    it('renders dropdowns to select database, schema, and table', () => {
```

Comment:
for a small test like this, perhaps merge with the following test to save runtime. 
Can also use `should('have.length')` instead of `.then( expect )`.